framework/assets/yii.js

```diff
@@ -284,67 +274,185 @@ window.yii = (function ($) {
             }
 
             var pairs = url.substring(pos + 1).split('#')[0].split('&'),
-                params = {},
-                pair,
-                i;
+                params = {};
 
-            for (i = 0; i < pairs.length; i++) {
-                pair = pairs[i].split('=');
+            for (var i = 0, len = pairs.length; i < len; i++) {
```

Comment:
Micro-optimization - calculate `len` only once, at the very beginning of the loop.